exercises/roman-numerals/introduction.md

```diff
@@ -0,0 +1,43 @@
+# Description
+
+The Romans were a clever bunch.
+They conquered most of Europe and ruled it for hundreds of years.
+They invented concrete and straight roads and even central heating.
+But one thing they never discovered though was the number zero.
+
+Their numbering system, known today as "Roman Numerals" uses letters - I, V, X, L, C, D, M.
+
+| I | V | X  | L  |  C  |  D  |  M   |
+|---|---|----|----|-----|-----|------|
+| 1 | 5 | 10 | 50 | 100 | 500 | 1000 |
+
+There are four basic rules for Roman Numerals:
+
+- When a larger letter precedes a smaller letter, the letters are added together. For example: in `VI`, `V` (5) is larger than `I` (1), so we add them together to get 6.
+- When a smaller letter precedes a larger letter, the letters are subtracted. For example: in `IV`, `I` (1) is smaller than `V` (5), so we subtract 1 from 5 to get 4.
+- When a letter is repeated multiple times, they get added. For example: with `XX` we add `X` (10) to `X` (10), to get 20.
+- The same letter cannot be used more than three times in succession.
+
```

Comment:
There is a missing rule: only the greatest lesser power-of-10 digit is allowed in subtractive position. That is: only IV, IX, XL, XC, CD, CM are allowed. (In Renaissance standard Roman numerals anyway.)

Reply:
> only the greatest lesser power-of-10 digit is allowed in subtractive position

Can you reword that so my brain can make sense of it pls? 😁

Reply:
I struggled to find a _nice_ summary, and still haven't found any. Possibly the most human-compatible way is to essentially list all possibilities:

> In subtractive position are allowed only:
> 
> - I before V, X
> - X before L, C
> - C before D, M
> 
> V, L, D are never allowed in subtractive position, nor are I, X when the following digit is 'too great', as in IC.

Reply:
So you can go only two steps back?

Reply:
> So you can go only two steps back?

I'd rather say you can "go back" to the previous power of 10 (from C or L to X, from X or V to I).